scripts/start_vm_qemu.sh

```diff
@@ -8,10 +8,11 @@ qemu-system-x86_64 \
     -m ${MEMORY:=2096} \
     -smp cores=2 \
     -nographic \
+    -enable-kvm \
```

Comment:
care to add it as optional? I use it in codespaces and there is no kvm in there :see_no_evil:

Reply:
enabled by default sounds perfectly fine too. we can also set an env var directly when creating the devcontainer here to turn off KVM https://github.com/kairos-io/kairos/blob/master/.devcontainer/post-create.sh

but yeah, don't want it to make it complex... otherwise we could also detect if we have /dev/kvm?

Reply:
done